apisix/plugins/echo.lua

```diff
@@ -70,4 +102,23 @@ function _M.body_filter(conf, ctx)
     ngx.arg[2] = true
 end
 
+function _M.access(conf, ctx)
+    local auth_header = core.request.header(ctx, "Authorization")
+
+    if conf.auth_header == "Authorization" and conf.auth_value == "userpass" then
+            return 200, "authorized body"
+    end
```

Comment:
`local auth_header` is it useless?

Reply:
@agile6v  wdyt ? 

I think at least we should showcase the workflow to pass auth header and then based on it give access in the access phase

Reply:
Yes. I have communicated with @Akayeshmantha on slack. And then review when the changes are completed?

Reply:
ok, got it.